app/cisc/sshconfig.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+/*
+A very basic ssh-config file reader and writer. It can handle comments in
+front of the 'Host' commands. When reading and writing, it is not guaranteed
+that the file stays the same.
+- sections always start with comment followed by "Host"
+- the configuration-part of a host has to start with a tab
+*/
+
+// SSHConfig holds all host-definitions
+type SSHConfig struct {
+	Host []*SSHHost
+}
+
+// NewSSHConfig takes a string which represents the ssh-config files, parses
```

Comment:
No, its `.ssh/config`, which can do a lot more. I also want to be able to keep the comments before an entry, as I would like to put some information for `cisc` in there (as they would be incompatible with SSH). So I wrote this library.